source/configure/database-configuration-settings.rst

```diff
@@ -81,7 +81,7 @@ Maximum open connections
 *Available in legacy Enterprise Edition E10/E20*
 
 +--------------------------------------------------------+------------------------------------------------------------------+
-| The maximum number of idle connections held open       | - System Config path: **Environment > Database**                 |
+| The maximum number of open connections held open       | - System Config path: **Environment > Database**                 |
```

Comment:
Thank you, @DragorWW! Please remove "held open" since it's redundant and not adding value.